torch/_inductor/ir.py

```diff
@@ -93,6 +93,8 @@ def _check_tensorbox(node):
             (
                 DynamicScalar,
                 TensorBox,
+                MultiOutput,
```

Comment:
This change is bad, because everything should be TensorBoxes

Reply:
hmm, now that you pointed out the above (return type) isn't it valid to return tuple(tensor) which is kind of what MultiOutput is?